rust/tw_keypair/src/ecdsa/canonical.rs

```diff
@@ -11,59 +11,62 @@
 //!
 //! **Not production ready**
 
-use crate::secp256k1::{PrivateKey, Signature};
-use crate::Error;
-use k256::ecdsa::hazmat::{bits2field, DigestPrimitive, SignPrimitive};
-use k256::elliptic_curve::generic_array::ArrayLength;
-use k256::elliptic_curve::subtle::{Choice, ConditionallySelectable, ConstantTimeEq};
-use k256::elliptic_curve::{Curve, FieldBytesEncoding, PrimeField};
+#![allow(dead_code)]
```

Comment:
Why this?

Reply:
`canonical` module contains a custom implementation of the ECDSA signing - Proof of Concept. I think it's worth to wait for an audit to validate if everything is implemented correctly.
None of the indets from `canonical` is exported publicly, so I decided to mark it with `allow(dead_code)` to check if the module is compiled correctly, but to ignore the warnings.